package.json

```diff
@@ -1,47 +1,91 @@
+    "@types/chai": "^4.2.20",
+    "@types/mocha": "^8.2.3",
+    "@typescript-eslint/eslint-plugin": "^4.26.0",
+    "@typescript-eslint/parser": "^4.28.1",
     "chai": "^4.3.4",
-    "chai-as-promised": "^7.1.1",
-    "dotenv": "^10.0.0",
-    "eslint": "^7.10.0",
-    "ethereum-waffle": "^3.4.0",
-    "ethers": "^5.5.1",
-    "hardhat": "^2.6.7",
-    "mocha": "^7.2.0"
+    "eslint": "^7.25.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-json": "^3.0.0",
+    "eslint-plugin-prettier": "^3.4.0",
+    "eslint-plugin-unused-imports": "^1.1.1",
+    "ethereum-waffle": "^3.3.0",
+    "ethers": "^5.4.4",
+    "ethlint": "^1.2.5",
+    "hardhat": "^2.6.0",
+    "hardhat-deploy": "^0.7.11",
+    "hardhat-gas-reporter": "^1.0.4",
+    "hardhat-spdx-license-identifier": "^2.0.3",
+    "husky": "^7.0.1",
+    "lint-staged": "^11.0.0",
+    "prettier": "^2.3.0",
+    "prettier-plugin-solidity": "^1.0.0-beta.11",
+    "shelljs": "^0.8.4",
+    "solhint": "^3.3.6",
+    "solhint-plugin-prettier": "0.0.5",
+    "solidity-coverage": "^0.7.16",
+    "solidity-docgen": "^0.5.13",
+    "ts-generator": "^0.1.1",
+    "ts-node": "^10.0.0",
+    "typechain": "^5.1.1",
+    "typescript": "^4.3.5"
   },
-  "eslintConfig": {
-    "extends": "eslint:recommended",
-    "parserOptions": {
-      "ecmaVersion": 2018,
-      "sourceType": "module"
-    },
-    "env": {
-      "es6": true,
-      "node": true
-    },
-    "rules": {
-      "semi": "error"
+  "scripts": {
+    "lint": "npm run lint:ts && npm run lint:sol && npm run lint:json",
+    "lint:fix": "npm run lint:fix:ts && npm run lint:fix:sol && npm run lint:fix:json",
+    "lint:ts": "eslint --ext ts test/ && eslint --ext ts deploy/ && prettier --list-different {test,deploy}/**/*.ts",
+    "lint:fix:ts": "eslint --ext ts --fix test/ && eslint --ext ts --fix deploy/ && prettier --write {test,deploy}/**/*.ts",
+    "lint:json": "eslint --ext json deployments/ && prettier --list-different deployments/**/*.json",
+    "lint:fix:json": "eslint --ext json deployments/ && prettier --write deployments/**/*.json",
+    "lint:sol": "solhint 'contracts/**/*.sol'",
+    "lint:fix:sol": "prettier --write 'contracts/**/*.sol' && solhint 'contracts/**/*.sol' --fix",
+    "build": "hardhat compile",
+    "test": "hardhat test",
+    "coverage": "export CODE_COVERAGE=true && hardhat coverage --temp ./build/artifacts ; unset CODE_COVERAGE",
+    "deploy": "hardhat deploy",
+    "start": "hardhat node --show-accounts",
+    "fork": "export FORK_MAINNET=true && hardhat node --fork-deployments mainnet --as-network localhost --show-accounts ; unset FORK_MAINNET",
+    "prepare": "npm run build"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
     }
+  },
+  "config": {
+    "eslintPaths": "test/ deploy/"
+  },
+  "keywords": [
+    "cryptocurrency",
+    "interoperability",
+    "bitcoin",
+    "ethereum",
+    "tbtc",
+    "defi"
+  ],
+  "author": "Satoshi Nakamoto 🤪",
```

Comment:
Should we change the author to be `Snowball` ?

Reply:
ill remove it